graveler/graveler.go

```diff
@@ -1105,15 +1115,16 @@ func (g *Graveler) Revert(ctx context.Context, repositoryID RepositoryID, branch
 		if err != nil {
 			return "", fmt.Errorf("set branch: %w", err)
 		}
-		return commitID, nil
+		return &CommitIDAndSummary{commitID, summary}, nil
 	})
+	c := res.(*CommitIDAndSummary)
 	if err != nil {
-		return "", err
+		return "", DiffSummary{}, err
 	}
-	return commitID.(CommitID), nil
+	return c.ID, c.Summary, nil
```

Comment:
Usually, I call the cast after the error check in case somebody will return just an error it will not panic.

Reply:
:-) Thanks!